core/testdata/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testdata contains data and data types for interoperability testing.
```

Comment:
You'd usually only really put the doc for a package in a separate file if the package description is getting a bit lengthy.  Couldn't this package doc go in testdata/types.go?

Reply:
Good idea.